README.md

```diff
@@ -53,7 +55,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
     <td align="center"><a href="http://faheempatel.com"><img src="https://avatars.githubusercontent.com/u/458311?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Faheem Patel</b></sub></a><br /><a href="https://github.com/hauptrolle/chakra-templates/commits?author=faheempatel" title="Code">💻</a></td>
     <td align="center"><a href="https://github.com/k-utsumi"><img src="https://avatars.githubusercontent.com/u/13148112?v=4?s=100" width="100px;" alt=""/><br /><sub><b>k-utsumi</b></sub></a><br /><a href="https://github.com/hauptrolle/chakra-templates/commits?author=k-utsumi" title="Code">💻</a></td>
     <td align="center"><a href="https://github.com/it-nalon"><img src="https://avatars.githubusercontent.com/u/8491676?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Marco Nalon</b></sub></a><br /><a href="https://github.com/hauptrolle/chakra-templates/commits?author=it-nalon" title="Code">💻</a></td>
-    <td align="center"><a href="https://github.com/Arithmetics"><img src="https://avatars.githubusercontent.com/u/25254098?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Brock Tillotson</b></sub></a><br /><a href="https://github.com/hauptrolle/chakra-templates/commits?author=Arithmetics" title="Code">💻</a></td>
+    <td align="center"><a href="https://github.com/Arithmetics"><img src="https://avatars.githubusercontent.com/u/25254098?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Brock Tillotson</b></sub></a><br /><a href="https://github.com/hauptrolle/chakra-templates/commits?author=Arithmetics" title="Code">💻</a></td>    <td align="center"><a href="https://github.com/rithulkamesh"><img src="https://avatars.githubusercontent.com/u/45348760?v=4" width="100px;" alt=""/><br /><sub><b>Rithul Kamesh</b></sub></a><br /><a href="https://github.com/hauptrolle/chakra-templates/commits?author=rithulkamesh" title="Code">💻</a></td>
   </tr>
 </table>
 
```

Comment:
Please do not modify this file by your own. I will add you after the MR is done via the allcontributors bot

Reply:
Restored